src/accelerator.rkt

```diff
@@ -137,11 +142,11 @@
         (cons (pattern-substitute (rule-output rule) bindings) bindings)
         #f)))
 
-(define-accelerator-operator expm1 (real) real (lambda (x) (- (exp x) 1)))
-(define-accelerator-operator log1p (real) real (lambda (x) (log (+ 1 x))))
-(define-accelerator-operator hypot (real real) real (lambda (x y) (sqrt (+ (* x x) (* y y)))))
-(define-accelerator-operator fma (real real real) real (lambda (x y z) (+ (* x y) z)))
-(define-accelerator-operator erfc (real) real (lambda (x) (- 1 (erf x))))
+(define-accelerator-operator expm1 (real) real (lambda (x) (- (exp x) 1)) ival-expm1)
+(define-accelerator-operator log1p (real) real (lambda (x) (log (+ 1 x))) ival-log1p)
+(define-accelerator-operator hypot (real real) real (lambda (x y) (sqrt (+ (* x x) (* y y)))) ival-hypot)
+(define-accelerator-operator fma (real real real) real (lambda (x y z) (+ (* x y) z)) ival-fma)
+(define-accelerator-operator erfc (real) real (lambda (x) (- 1 (erf x))) ival-erfc)
```

Comment:
Cool, we can do this for now and then once we move everything into Rival we'll get rid of this, as discussed.